webapp/src/Twig/TwigExtension.php

```diff
@@ -150,6 +150,7 @@ public function getGlobals()
             'refresh_flag' => $refresh_flag,
             'icat_url' => defined('ICAT_URL') ? ICAT_URL : null,
             'external_ccs_submission_url' => $this->config->get('external_ccs_submission_url'),
+            'show_public_stats' => $this->config->get('show_public_stats'),
```

Comment:
Why is this extra global needed? The template can just see if stats is filled?

Reply:
You are right. This was needed before when it was a separate page. I will change it